website/docs/d/nsxt_network_dhcp_binding.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_nsxt_network_dhcp_binding"
+sidebar_current: "docs-vcd-datasource-nsxt-network-dhcp-binding"
+description: |-
+  Provides a data source to read NSX-T Org VDC network DHCP bindings.
+---
+
+# vcd\_nsxt\_network\_dhcp\_binding
+
+Provides a data source to read NSX-T Org VDC network DHCP bindings.
+
+-> This data source requires VCD 10.3.1+
+
+## Example Usage
+
+```hcl
+data "vcd_nsxt_network_dhcp" "pools" {
+  org = "cloud"
+
+  org_network_id = data.vcd_network_routed_v2.parent.id
+}
+data "vcd_nsxt_network_dhcp_binding" "binding1" {
+  org            = "cloud"
+  org_network_id = data.vcd_nsxt_network_dhcp.pools.id
+  name           = "Binding-one"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional) The name of organization. Optional if defined at provider level
+* `org_network_id` - (Required) The ID of an Org VDC network. **Note**  `.id` field of
```

Comment:
Could we use brackets around `.id field` just like in the resource so it looks cleaner?

Reply:
LOL, we have collided here in taste :) As I have requested to remove the parenthesis here:
https://github.com/vmware/terraform-provider-vcd/pull/1039#discussion_r1158306032

Reply:
I have changed it to Linas view, but we can discuss further

Reply:
It's fine by me to leave as it is right now, just should be consistent across two places

Reply:
At least I like how it looks right now:

![Screenshot 2023-04-07 at 09 38 21](https://user-images.githubusercontent.com/11330689/230555553-e6265a2f-c40b-4b80-8817-398ff1a593eb.png)